src/app/images/components/ImagesTable/ImagesTable.tsx

```diff
@@ -150,6 +171,14 @@ const generateResourceRow = ({
       { content: resource.title, className: "release-col" },
       { content: resource.arch, className: "arch-col" },
       { content: resource.size, className: "size-col" },
+      {
+        content: (
+          <EphemeralSupportIcon
+            supports_ephemeral={resource.canDeployToMemory}
```

Comment:
We use camelCase for all props across maas-ui. I'd suggest to update this to be consistent. Also - have you considered using a single prop (e.g. only canDeployToMemory instead of introducing additional support_ephemeral)?

Reply:
Good spot - that was a leftover from the boilerplate code, before the backend changes landed. Will update